Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocListCssClassDefinition.cs

```diff
@@ -161,6 +161,20 @@ public virtual string TitleCellSelector
       get { return "bocListTitleCellAllRowsSelector"; }
     }
 
+    /// <summary> Gets the CSS-Class applied to the cell in the <see cref="BocList"/>'s data rows that contains the edit mode controls. </summary>
+    /// <remarks> Class: <c>bocListDataCellEditMode</c> </remarks>
+    public virtual string DataCellEditMode
+    {
+      get { return "bocListDataCellEditMode"; }
+    }
+
+    /// <summary> Gets the CSS-Class applied to the cell in the <see cref="BocList"/>'s title row that contains the edit mode controls. </summary>
+    /// <remarks> Class: <c>bocListTitleCellEditMode</c> </remarks>
+    public virtual string TitleCellEditMode
+    {
+      get { return "bocListTitleCellEditMode"; }
+    }
```

Comment:
I think we should add "Buttons" (bocListTitleCellEditModeButtons" and "bocListDataCellEditModeButtons") just to make it clearer when reading the css.